### ISSUE TITLE
Suggestion: Harvest/Death gains display for garden

### ISSUE THREAD

**Issue by Rodrigo-P** [Aktanusa/CookieMonster#293]
I saw some previous topics covering this, but they're quite old.

To explain the feature I'll use Chocoroot as an example:
When harvesting the cookies gained are similar to a lucky golden cookie, the minimum between 3 min of Cps and 3% of your current cookies. Is it possible to, when you hover the plant, show the amount of cookies you'll gain, the maximum you could gain and how many cookies you need stored to get this maximum.

Basically the Lucky Cookies tab that's inside the status, but for the plants and inside their info-box.

**Comment by Aktanusa** [comment]
It's possible.  Good idea.  Too lazy ATM to see which issue is similar, lol.

**Comment by DanielNoord** [comment]
This is a duplicate of #277 and #183
